react/utils/index.tsx

```diff
@@ -188,3 +188,14 @@ export function orderSplitMessage({
     takeaways,
   })
 }
+
+export function parseBankInvoiceUrl(url: string) {
+  const isEncrypted = url.match(/(\*.\*.)+\*\w\*/g)
+
+  if (!isEncrypted) return url
+  if (!window || !window.location) return ''
+
+  return `/login?returnUrl=${encodeURIComponent(
```

Comment:
I think there's rootPath:
https://vtex.slack.com/archives/CD0C3M3MJ/p1565129756184500

For dealing with stores like Samsung AR

So you might need to append some string here